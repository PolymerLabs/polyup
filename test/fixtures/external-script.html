<link rel='import' href='../polymer/polymer.html'>

<polymer-element name='has-external-script'>
  <template></template>
  <script src='external-script.js'></script>
</polymer-element>

<polymer-element name='also-in-that-external-script'>
</polymer-element>

<script src='an-unaffiliated-script.js'></script>

<polymer-element name='has-its-own-script'>
  <script>
    Polymer('has-its-own-script', {});
  </script>
</polymer-element>

<polymer-element name='in-the-above-script'></polymer-element>


<script src="//google.com/jquery.js"></script>
