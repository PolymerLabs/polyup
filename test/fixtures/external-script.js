Polymer({});

Polymer('also-in-that-external-script', {});
