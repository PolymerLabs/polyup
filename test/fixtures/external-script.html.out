<link rel="import" href="../polymer/polymer.html">

<dom-module id="has-external-script">
  <template></template>
  <script src="external-script.js"></script>
</dom-module>

<dom-module id="also-in-that-external-script">
</dom-module>

<script src="an-unaffiliated-script.js"></script>

<dom-module id="has-its-own-script">
  <script>
    Polymer({ is: 'has-its-own-script' });
  </script>
</dom-module>

<dom-module id="in-the-above-script">
</dom-module>


<script src="//google.com/jquery.js"></script>
